### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by divan**
Added basic info on build and testing to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,34 @@
...
+Integration tests and server require `statusd-data` dir to store synched blockchain and other stuff, so we use `build/env.sh` script before running most of the commands in Makefile (like `go test` or running the server)
+
+# Build
+There are two main modes status-go can be built:
+ - standalone server
+ - library to link for Android or iOS
+
+Use following Makefile commands:
+
+`make statusgo` (builds binary into `build/bin/statusd`)
```
Comment (themue) [inline README.md]: Needs a dashed list here for correct rendering.

File: README.md [hunk 2]
```diff
@@ -12,3 +12,34 @@
 - [How To Build](https://github.com/status-im/status-go/wiki/Build-Process-Explained)
```
Comment (themue) [inline README.md]: Leads to non-existing document in Git wiki while the existing one at https://github.com/status-im/status-go/wiki/Building is outdated.

Comment (divan) [reply README.md]: Oh, right. I didn't touch that part. Will link it to notion.so as it's most update to the date.

File: README.md (hunk 1 above)
Comment (divan) [reply README.md]: Ouch, thanks!

**Comment by divan** [comment]
> Would like to see README only as introduction

That's exactly what this short README aims to accomplish. Basic building/testing commands arent' subject to change often, but gives enough information to start with for anyone looking at the repo in github. Empty READMEs always look suspicious.

File: README.md [hunk 3]
```diff
@@ -9,9 +9,9 @@
```
Comment (themue) [inline README.md]: Would you mind this change to also move the tags form the headline to the other two for Travis and GoDoc?

File: README.md [hunk 4]
```diff
@@ -1,14 +1,45 @@
...
+[![GoDoc](https://godoc.org/github.com/status-im/status-go?status.svg)](https://godoc.org/github.com/status-im/status-go) [![Master Build Status](https://img.shields.io/travis/status-im/status-go/master.svg?label=build/master)](https://github.com/status-im/status-go/tree/master) [![Develop Build Status](https://img.shields.io/travis/status-im/status-go/develop.svg?label=build/develop)](https://github.com/status-im/status-go/tree/develop)
+
+# Docs
 
+- [How To Build](https://www.notion.so/status/Building-status-go-f6b827dd1302436ba0575f4c543a352e)
+- [Notes on Bindings](https://www.notion.so/status/Binding-notes-344f30ce0f2845a2b43e2de70931284a)
+- [Status-go docs](https://www.notion.so/status/status-go-4fbe361e8e75484abeadadc80dd4dcdc)
 
+# Intro
+status-go is a underlying part of status mobile app and handles blockchain and Status accounts related code.
```
Comment (adambabik) [inline README.md]: Typo: `an underlying`.

`related code.` at the end is suppose to be here?

File: README.md [hunk 5]
```diff
@@ -1,14 +1,45 @@
...
 
+It's written in Go and requires Go 1.8 or above.
 
-# Docs
+It uses Makefile to do most common actions. See `make help` output for available commands.
+
+Integration tests and server require `statusd-data` dir to store synched blockchain and other stuff, so we use `build/env.sh` script before running most of the commands in Makefile (like `go test` or running the server)
```
Comment (adambabik) [inline README.md]: Not sure it's relevant here and also not sure how exactly it works with tests. There is also `.ethereumtest` directory with keys and blockchain data.

**Pull request merged**